automatic/service-fabric-sdk/service-fabric-sdk.nuspec

```diff
@@ -18,13 +18,13 @@ Service Fabric is a distributed systems platform that makes it easy to package,
     <licenseUrl>https://github.com/Microsoft/service-fabric/blob/master/LICENSE</licenseUrl>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <iconUrl>https://cdn.jsdelivr.net/gh/darinegan/chocolatey-packages@latest/icons/service-fabric.png</iconUrl>
-    <releaseNotes>[Microsoft Azure Service Fabric 10.0 Release Notes](https://github.com/microsoft/service-fabric/blob/master/release_notes/Service_Fabric_ReleaseNotes_10.md)</releaseNotes>
+    <releaseNotes>[Microsoft Azure Service Fabric Release Notes](https://github.com/microsoft/service-fabric/tree/master/release_notes)</releaseNotes>
```

Comment:
Let's update the URL to the specific version release notes.

Reply:
This is updated now